resources/l10n/fi/LC_MESSAGES/loot.po

```diff
@@ -1,350 +1,350 @@
-msgstr "Tiedosto"
+#: src/gui/qt/main_window.cpp:643
+msgid "&File"
+msgstr "&Tiedosto"
 
-#: src/gui/qt/main_window.cpp:590
-msgid "Settings..."
-msgstr "Asetukset..."
+#: src/gui/qt/main_window.cpp:644
+msgid "&Settings..."
+msgstr "&Asetukset..."
 
-#: src/gui/qt/main_window.cpp:591
-msgid "Backup LOOT Data"
-msgstr "Varmuuskopioi LOOTin tiedot"
+#: src/gui/qt/main_window.cpp:645
+msgid "&Backup LOOT Data"
+msgstr "&Varmuuskopioi LOOTin tiedot"
 
-#: src/gui/qt/main_window.cpp:592
-msgid "Open LOOT Data Folder"
-msgstr "Avaa LOOTin tiedostokansio"
+#: src/gui/qt/main_window.cpp:646
+msgid "&Open LOOT Data Folder"
+msgstr "Avaa &LOOTin tiedostokansio"
 
-#: src/gui/qt/main_window.cpp:593
-msgid "Quit"
-msgstr "Poistu"
+#: src/gui/qt/main_window.cpp:647
+msgid "&Quit"
+msgstr "&Poistu"
 
-#: src/gui/qt/main_window.cpp:595
-msgid "Game"
-msgstr "Peli"
+#: src/gui/qt/main_window.cpp:649
+msgid "&Game"
+msgstr "P&eli"
 
-#: src/gui/qt/main_window.cpp:596
-msgid "Edit Groups..."
-msgstr "Muokkaa ryhmiä..."
+#: src/gui/qt/main_window.cpp:650
+msgid "&Edit Groups..."
+msgstr "&Muokkaa ryhmiä..."
 
-#: src/gui/qt/main_window.cpp:597
-msgid "Search Cards..."
-msgstr "Hae korteista..."
+#: src/gui/qt/main_window.cpp:651
+msgid "Searc&h Cards..."
+msgstr "&Hae korteista..."
 
-#: src/gui/qt/main_window.cpp:598
-msgid "Copy Load Order"
-msgstr "Kopioi järjestys"
+#: src/gui/qt/main_window.cpp:652
+msgid "Copy &Load Order"
+msgstr "Kopioi &järjestys"
 
-#: src/gui/qt/main_window.cpp:599
-msgid "Copy Content"
-msgstr "Kopioi sisältö"
+#: src/gui/qt/main_window.cpp:653
+msgid "&Copy Content"
+msgstr "&Kopioi sisältö"
 
-#: src/gui/qt/main_window.cpp:600
-msgid "Refresh Content"
-msgstr "Päivitä sisältö"
+#: src/gui/qt/main_window.cpp:654
+msgid "&Refresh Content"
+msgstr "Päivitä &sisältö"
 
-#: src/gui/qt/main_window.cpp:601
-msgid "Redate Plugins..."
-msgstr "Muuta lisäosien päiväykset..."
+#: src/gui/qt/main_window.cpp:655
+msgid "Redate &Plugins..."
+msgstr "Muuta &lisäosien päiväykset..."
 
-#: src/gui/qt/main_window.cpp:602
-msgid "Fix Ambiguous Load Order"
-msgstr "Korjaa epäselvä järjestys"
+#: src/gui/qt/main_window.cpp:656
+msgid "&Fix Ambiguous Load Order"
+msgstr "Korjaa &epäselvä järjestys"
 
-#: src/gui/qt/main_window.cpp:603
-msgid "Clear All User Metadata..."
-msgstr "Poista kaikki muokatut metatiedot..."
+#: src/gui/qt/main_window.cpp:657
+msgid "Clear All &User Metadata..."
+msgstr "&Poista kaikki muokatut metatiedot..."
 
-#: src/gui/qt/main_window.cpp:605
-msgid "Plugin"
-msgstr "Lisäosa"
+#: src/gui/qt/main_window.cpp:659
+msgid "&Plugin"
+msgstr "&Lisäosa"
 
-#: src/gui/qt/main_window.cpp:606
-msgid "Copy Plugin Name"
-msgstr "Kopioi lisäosan nimi"
+#: src/gui/qt/main_window.cpp:660
+msgid "Copy &Plugin Name"
+msgstr "&Kopioi lisäosan nimi"
 
-#: src/gui/qt/main_window.cpp:607
-msgid "Copy Card Content"
-msgstr "Kopioi kortin sisältö"
+#: src/gui/qt/main_window.cpp:661
+msgid "Copy &Card Content"
+msgstr "Kopioi &kortin sisältö"
```

Comment:
I think this and the `&Kopioi sisältö` and `Kopioi &kortin sisältö` mnemonic are ambiguous as they all appear in the Game menu and shortcuts aren't case-sensitive.

Reply:
This is under Plugins menu, unless moved in .4, sisältö is under Game.